README.md

```diff
@@ -190,27 +190,26 @@ If a track implements an exercise for which test data exists, the exercise _must
 A `tests.toml` file for a track's `two-fer` exercise looks like this:
 
 ```toml
-[canonical-tests]
+[19709124-b82e-4e86-a722-9e5c5ebf3952]
+description = "no name given"
 
-# no name given
-"19709124-b82e-4e86-a722-9e5c5ebf3952" = true
+[3451eebd-123f-4256-b667-7b109affce32]
+description = "a name given"
+include = false
 
-# a name given
-"3451eebd-123f-4256-b667-7b109affce32" = true
-
-# another name given
-"653611c6-be9f-4935-ab42-978e25fe9a10" = false
+[653611c6-be9f-4935-ab42-978e25fe9a10]
+description = "another name given"
 ```
 
-In this case, the track has chosen to implement two of the three available tests.
+In this case, the track has chosen to implement two of the three available tests (`include = true` is the default and is omitted).
 
 If a track uses a _test generator_ to generate an exercise's test suite, it _must_ use the contents of the `tests.toml` file to determine which tests to include in the generated test suite.
 
 ### Track Test Data Tooling
 
-To make it easy to keep the `tests.toml` up to date, tracks can use the [`canonical_data_syncer` application](https://github.com/exercism/canonical-data-syncer). This application is a small, standalone binary that will compare the tests specified in the `tests.toml` files against the tests that are defined in the exercise's canonical data. It then interactively gives the maintainer the option to include or exclude test cases that are currently missing, updating the `tests.toml` file accordingly.
+To make it easy to keep the `tests.toml` up to date, tracks can use the [`configlet` application](https://github.com/exercism/configlet)'s `sync` command. This command will compare the tests specified in the `tests.toml` files against the tests that are defined in the exercise's canonical data. It then interactively gives the maintainer the option to include or exclude test cases that are currently missing, updating the `tests.toml` file accordingly.
```

Comment:
Non-blocking comment: the default `sync` behavior will stop being interactive in the future. We should remember to update this later.

Reply:
We should